tests/Tests/Services/TrustedUserServiceTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * TrustedUserServiceTest.php
+ * @package openemr
+ * @link      http://www.open-emr.org
+ * @author    Stephen Nielson <stephen@nielson.org>
+ * @copyright Copyright (c) 2021 Stephen Nielson <stephen@nielson.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Tests\Services;
+
+use OpenEMR\Services\TrustedUserService;
+use OpenEMR\Services\UserService;
+use PHPUnit\Framework\TestCase;
+use Ramsey\Uuid\Uuid;
+
+class TrustedUserServiceTest extends TestCase
+{
+    private $clientId;
+    const TRUSTED_CLIENT_TAG = '-trusted-user-test';
+
+    public function setUp(): void
+    {
+    }
+
+    public function tearDown(): void
+    {
+        // clean up any database records we made
+        sqlQueryNoLog("DELETE FROM `oauth_trusted_user` WHERE `code` LIKE '%"
+            . self::TRUSTED_CLIENT_TAG . "%'");
+    }
+
+    /**
+     * We had api authorization issues where null user id's were not returning as trusted user's
```

Comment:
does this function all need to be reworded to system user is rather than null user id?

Reply:
Yep, it did need to and I fixed it.